Top Secret: update hints.md (#1082)
exercism/elixir: exercises/concept/top-secret/.docs/hints.md

- The operation is the first element in a three-element AST node tuple.
- You can ignore the second element in the tuple in this exercise completely.
- The third element in the tuple is the argument list of the operation that defines the function.

> - The first element on that list is the function's name with any function parameters present in an embedded seperate list, and the second element is the function's body.

Reviewer: Neat!
It's a bit long to read, maybe something a bit shorter?
"The first element on that list is a tuple with the function's name and arguments, and the second is..."

> - The first element on that list is the function's name with any function parameters present in an embedded seperate list, and the second element is the function's body.

Reviewer: ```suggestion
- The first element on that list is a tuple with the function's name and arguments, and the second element is the function's body.
```
